Makefile

```diff
@@ -286,6 +286,7 @@ kind-delete-cluster:
 install-crds:
 	@echo installing crds
 	kubectl apply -f deploy/crds/policy.open-cluster-management.io_iampolicies.yaml
+	kubectl apply -f https://raw.githubusercontent.com/stolostron/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies.yaml
```

Comment:
I'd say this should have a variable branch specified to replace `main`, but I don't think it really matters for this.